Documentation/OfflineGuide/README.md

```diff
@@ -27,3 +27,37 @@ const viewer = new Cesium.Viewer("cesiumContainer", {
   geocoder: false,
 });
 ```
+
+## 3D Tiles, glTF, and other static files
+
+Most other files loaded in CesiumJS, such as 3D Tiles or glTF, are static assets that do not require any server-side operations to load. However, since browsers commonly treat requests to load resources using the `file://` schema as cross-origin requests, it's reccomended that you set up a local server.
+
+1. Download and install [Node.js](https://nodejs.org/en/download/)
+
+2. At the command line, run
+
+   ```sh
+   npm install http-server -g
```

Comment:
Can we suggest using `npx` instead? I know I've run into permissions issues sometimes when installing packages globally on other systems.

Reply:
Is that a common instruction in getting started guides like this? I don't see it in any of the top npm modules like React, tslib, lodash, etc.

Reply:
> Is that a common instruction in getting started guides like this?

I am not sure if it's common. If `npm` is recommended more often then that's fine too.